jscomp/others/js_option.ml

```diff
@@ -59,5 +59,27 @@ let andThen f x =
   | None -> None 
   | Some x -> f x [@bs]
 
+let map f x =
+  match x with
+  | None -> None
+  | Some x -> Some (f x [@bs])
 
+let orValue a x =
+  match x with
+  | None -> a
+  | Some x -> x
```

Comment:
The stock OCaml `Option` module (bucklescript really needs to update) uses `default` for this exact same function:
```ocaml
let default d = function
| None -> d
| Some a -> a
```
Probably be good to add in the same names at the very least.